recipes/viral_consensus/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.0.4" %}
+
+package:
+  name: viral_consensus
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('viral_consensus', max_pin="x.x.x") }}
+
+source:
+  url: https://github.com/niemasd/ViralConsensus/archive/refs/tags/{{ version }}.tar.gz
+  sha256: d531311c8e5c6b689f4426a9e89cc00426733ae9d4ff71757a861ad6ddbf81f4 # curl -sL URL | openssl sha256
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - bzip2
+    - htslib
+    - libcurl
+    - make
+    - ncurses
+    - zlib
+  host:
+    - bzip2
+    - htslib
+    - libcurl
+    - ncurses
+    - zlib
+  run:
+    - bzip2
+    - htslib
+    - libcurl
+    - ncurses
+    - zlib
```

Comment:
Just `htslib` for `run:` or in fact you should probably leave this empty and the dependency will be automatically derived.

Reply:
Thanks! A similar tool (`ivar`) seems to have `htslib` under `run:`, but I'll try it without it and re-add if it breaks things

Reply:
In the past it was necessary, but more recently most libraries such as htslib propagate the right thing from `host:`.